### PR TITLE
Don't escape markup sensitive characters in <title>

### PR DESCRIPTION
**Pull request by majakomel**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Avoids broken titles like `The DEV &lt;&gt; Stackbit integration is in the running for a Product Hunt "Golden Kitty Award"` by replacing `.innerHTML` with `.textContent` which gets a raw copy of text node content.

❗️This bug only appears when articles are not accessed via internal navigation.

## Related Tickets & Documents
Closes https://github.com/thepracticaldev/dev.to/issues/5226

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

html_escape

### CODE REVIEW

**Comment by nickytonline** [comment]
Ahh, just realized `TITLE` vs `title` yields different results. But regardless, why are there two `<title />` tags for the `document`?

**Comment by rhymes** [comment]
@nickytonline is still an issue in your opinion? The article that prompted this issue related to this PR seems fine: https://dev.to/astrit/why-how-did-i-build-500-css-only-icons-library-a-life-story-34on

**Comment by Defman21** [comment]
@rhymes it is.

image

**Comment by rhymes** [comment]
@Defman21 my mistake! For some reason I thought the issue was about the title we display, not the HTML `<title>` tag :) Yep, still an issue :)

**Comment by maestromac** [comment]
Hmm, this could be a browser-specific issue. Of course we still need to address it

image

File: app/views/shell/_bottom.html.erb [hunk 1]
```diff
@@ -10,7 +10,7 @@
   <% if @shell %>
     <script defer>
       if (!document.title) {
-        document.title = document.getElementsByTagName("TITLE")[1].innerHTML
+        document.title = document.getElementsByTagName("TITLE")[1].textContent
```
[the remark this answers is not in the document]
Comment (maestromac) [reply app/views/shell/_bottom.html.erb]: I'm a little late to this but I don't think this is no longer an issue @nickytonline

**Pull request merged**